great_expectations/checkpoint/checkpoint.py

```diff
@@ -24,6 +25,9 @@ def __init__(
         data_context,
         checkpoint_config: Union[CheckpointConfig, dict],
     ):
+        # TODO why is DataContext not importable?
```

Comment:
I'm guessing circular dependency since DataContext had code that instantiates a Checkpoint. Did a quick check though and there's only one instantiation, the rest of occurrences are type hints.

Reply:
Resolved with some gross but effective hacks.